gio/src/dbus_node_info.rs

```diff
@@ -0,0 +1,38 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use std::ffi::CStr;
+
+use glib::translate::ToGlibPtr;
+
+use crate::{DBusInterfaceInfo, DBusNodeInfo};
+
+impl DBusNodeInfo {
+    pub fn path(self) -> &'static str {
+        let c_obj = self.to_glib_none().0;
```

Comment:
I'd make this one big `unsafe` block in this function as it's all related really

Reply:
Done